build/build.js

```diff
@@ -4,25 +4,95 @@ var semver = require('semver');
 var fs = require('fs');
```

Comment:
the build script has seen a lot of evolutions:
* proper option parsing with minimist
* a lot of information is fetched directly from the manifest
* the update file has been converted to the json format (see https://developer.mozilla.org/en-US/Add-ons/Updates) and most of its information come from the manifest